### DISCUSSION TITLE
LNA

### DISCUSSION THREAD

**Discussion by sefirot43** [AlexandreRouma/SDRPlusPlus#401] [Q&A]
Good morning, I am using a LNA 50 khz-100 mhz along with a rtl-sdr v3.
Does it makes sense to use rtl agc and/or tuner agc assuming the program somehow adjust by itself the agc?
Thank you.

**Comment by AlexandreRouma** [comment]
the AGCs on the RTL-SDR are quite bad and shouldn't be used. I only allow to use them because they're available in the library.
They were designed for high bandwidth signals like DVB-T and behave badly with other types of signal.
The best way to adjust the gain is to increase it slowly until the noise floor starts raising as well (aka the SNR of signals doesn't increase anymore) all while avoiding to clip the SDR (you'll notice if you see a lot of noise all of a sudden)

**Reply by sefirot43** [reply]
Thank you.
